modules/mysql/variables.tf

```diff
@@ -55,6 +55,12 @@ variable "activation_policy" {
   default     = "ALWAYS"
 }
 
+variable "availability_type" {
+  description = "The availability type for the master instance. Can be either `ZONAL` or `REGIONAL`."
+  type        = string
+  default     = "ZONAL"
```

Comment:
I recommend to set default to REGIONAL as it is new default value and in July it will be even not possible to change it to ZONAL, also, also this info probably worth adding to description

Reply:
Either REGIONAL or null but not ZONAL. Both values (regional and null) will be correct in the future as well.

Reply:
Fixed in c247027 by setting to `REGIONAL`, description also updated to reflect this.

> in July it will be even not possible to change it to ZONAL

It's not going to be possible to create non HA masters in CloudSQL?

Reply:
I think that null is the value to create non-ha CloudSQL